perses/rjmc/geometry.py

```diff
@@ -226,6 +227,11 @@ def _logp_propose(self, top_proposal, old_positions, beta, new_positions=None, d
             # DEBUG: Write growth stages
             from simtk.openmm.app import PDBFile
             prefix = '%s-%d-%s' % (self.pdb_filename_prefix, self.nproposed, direction)
+            self._proposal_pdbfile = open("%s-proposal.pdb" % prefix, 'w') # PDB file for proposal probabilities
```

Comment:
I've been meaning to remove those PDB-writing blocks and replace them with NetCDF-writing blocks. Should we do that here or in a separate PR?